js/imageeditor/Index.svelte

```diff
@@ -31,7 +31,7 @@
 	};
 	export let label: string;
 	export let show_label: boolean;
-	export let show_download_button: boolean;
+	export let buttons: string[] | null = null;
```

Comment:
I think we should add `buttons` to the JSON component. There is a `show_copy_button` prop in the svelte file that's not controlled from python (oversight)